internal/databases/mysql/mysql.go

```diff
@@ -27,7 +27,7 @@ const (
 )
 
 type Uploader struct {
-	*internal.Uploader
+	*internal.WalUploader
```

Comment:
Mysql doesn't have WAL

Reply:
But it needs WalUploader, see other comment.